05/app.js

```diff
@@ -1 +1,14 @@
-const numbers = [1, 2, 3, 4, 5, 6, 7];
\ No newline at end of file
+const numbers = [1, 2, 3, 4, 5, 6, 7];
+const evenNumbers = numbers.filter( (number) => {
+	if (number % 2 === 0) {
+		return number;
```

Comment:
Ten element może powodować błąd. Co jeśli `number` będzie `0`? Wtedy ten element nie zostanie dołączony do tablicy.
Tutaj najlepszym rozwiązanie będzie nie korzystanie z `if`, a napisanie tylko: `return number % 2 === 0` Tylko wtedy jak nie ma reszty to dany element z tablicy dodajemy do nowo utworzonej. 

PS. Pomijamy temat czy 0 to parzysta czy nie :D